Documentation/RELEASE_NOTES.md

```diff
@@ -46,53 +46,53 @@
 
 This release was tested according to the following test matrix. Other combinations and versions have been tested in previous releases of MetroAE and are likely to work. We encourage you to test this in your lab before you apply it in production.
 
-Workflow | Target Server | Version
--------- | -------- | --------
-Install | KVM | Geo-redundant 20.10.R1
-Install | KVM | HA 5.4.1
-Install | KVM | HA 6.0.3
-Install | KVM | HA 20.10.R1
-Install | KVM | Active-Standby ES 20.10.R1
-Install | KVM | HA IPv6 20.10.R1
-Install | KVM | SA 5.4.1
-Install | KVM | SA 6.0.3
-Install | KVM | SA 20.10.R1
-Install | KVM | Add VSC pair 20.10.R1
-Install | KVM | SA CPU pinning 20.10.R1
-Install | KVM | SA VNS end-to-end 20.10.R1
-Install | KVM | SA VNS multi-uplinks 20.10.R1
-Install | KVM | SA feature tests 20.10.R1
-Install | KVM | SA IPv6 20.10.R1
-Install | KVM | SA Terraform 20.10.R1
-Install | KVM | SA via Container 20.10.R1
-Install | OpenStack | HA 5.3.2
-Install | OpenStack | HA 20.10.R1
-Install | OpenStack | SA 5.3.2
-Install | OpenStack | SA 20.10.R1
-Install | vCenter | HA w/VCIN 20.10.R1
-Install | vCenter | HA w/Hybrid VCIN 20.10.R1
-Install | vCenter | HA 5.4.1
-Install | vCenter | HA 6.0.3
-Install | vCenter | HA 20.10.R1
-Install | vCenter | HA Custom passwords 20.10.R1
-Install | vCenter | SA 5.4.1
-Install | vCenter | SA 6.0.3
-Install | vCenter | SA 20.10.R1
-Install | vCenter | SA Custom passwords 20.10.R1
-Upgrade | KVM | Geo-redundant 6.0.3-20.10.R1
-Upgrade | KVM | HA 5.4.1-6.0.3
-Upgrade | KVM | HA inplace 6.0.3-6.0.5
-Upgrade | KVM | HA 6.0.3-20.10.R1
-Upgrade | KVM | HA hardened 6.0.3-20.10.R1
-Upgrade | KVM | HA inplace 20.10.R1-20.10.R2
-Upgrade | OpenStack | HA 6.0.3-20.10.R1
-Upgrade | OpenStack | SA 6.0.3-20.10.R1
-Upgrade | vCenter | HA 6.0.3-20.10.R1
-Upgrade | vCenter | HA hardened 6.0.3-20.10.R1
-Upgrade | vCenter | HA inplace 6.0.3-6.0.5
-Upgrade | vCenter | HA inplace 20.10.R1-20.10.R2
-Upgrade | vCenter | SA hardened 6.0.3-20.10.R1
-Upgrade | vCenter | SA 6.0.3-20.10.R1
-Wizard Install | KVM | SA via container 20.10.R1
-Wizard Install | KVM | SA via CSV 20.10.R1
-Wizard Upgrade | KVM | SA 6.0.3-20.10.R1
+Status  | Workflow | Target Server | Version
+------- | -------- | -------- | --------
+Passing | Install | KVM | Geo-redundant 20.10.R1
```

Comment:
Will this be updated everytime the job runs? How to we map the test result to the job? Maybe the version should be names as test name in that case?